codefori/vscode-ibmi
# Remove SEU colour support

### Changes

* Removes all logic tied to SEU/5250 colours. See #2057 for more information.
* Adds new logic to cleanup members if they contain bad bytes.

Closes #2056.

### Todo

* Add translation for cleanup UI question ✅
* Add test case ✅

### How to test this PR

Create a member using SQL:

<details>
<summary>Click me</summary>

```sql
CREATE OR REPLACE ALIAS ILEDITOR.LIAMA_QRPGLESRC_COLOURS for "LIAMA"."QRPGLESRC"("COLOURS");

delete from ILEDITOR.LIAMA_QRPGLESRC_COLOURS;

insert into ILEDITOR.LIAMA_QRPGLESRC_COLOURS (srcseq, srcdat, srcdta)
values 
  (01.00, 240805, '      // This illustrates 5250 attribute bytes.'),
  (02.00, 240805, '     '),
--(05.00, 240805, '     C*' concat x'XX' concat 'X''XX''' concat x'404020' concat  'X''XX'' GRN RI UL BL CS ND'),
  (03.00, 240805, '     C*' concat x'20' concat 'X''20''' concat x'404020' concat  'X''20'' GRN               '),
  (04.00, 240805, '     C*' concat x'24' concat 'X''24''' concat x'404020' concat  'X''24'' GRN    UL         '),
  (05.00, 240805, '     C*' concat x'25' concat 'X''25''' concat x'404020' concat  'X''25'' GRN RI UL         '),
  (06.00, 240805, '     C*' concat x'2E' concat 'X''2E''' concat x'404020' concat  'X''2E'' RED    UL BL      '),
  (07.00, 240805, '     C*' concat x'2F' concat 'X''2F''' concat x'404020' concat  'X''2F'' RED             ND'),
  (08.00, 240805, '     C*' concat x'30' concat 'X''30''' concat x'404020' concat  'X''30'' TRQ          CS   '),
  (09.00, 240805, '     C*' concat x'36' concat 'X''36''' concat x'404020' concat  'X''36'' YLW          CS   '),
  (10.00, 240805, '     C*' concat x'37' concat 'X''37''' concat x'404020' concat  'X''37'' YLW          CS ND'),
  (11.00, 240805, '     C*' concat x'3D' concat 'X''3D''' concat x'404020' concat  'X''3D'' PNK RI UL         '),
  (12.00, 240805, '     C*' concat x'3E' concat 'X''3E''' concat x'404020' concat  'X''3E'' BLU    UL         '),
  (13.00, 240805, '     C*' concat x'3F' concat 'X''3F''' concat x'404020' concat  'X''3F'' revert to default '),
  (14.00, 240805, '     '),
  (15.00, 240805, '       *inlr = *on;'),
  (16.00, 240805, '       return;')
```

</details>

Then open the member with Code for IBM i and expect to see the question for cleanup.

### Checklist
